src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,7 @@
+$card-border-color: #f3f3f3;
+$link-color: #060b35;
+$secondary-accent: #616070;
+$blue-accent: #00acdc;
```

Comment:
Variable name should show their purpose, for example this color change to red in the palette and in this case you have to change name in all places